plexapi/base.py

```diff
@@ -324,7 +331,7 @@ def findItems(self, data, cls=None, initpath=None, rtag=None, **kwargs):
         if rtag:
             data = next(utils.iterXMLBFS(data, rtag), [])
         # loop through all data elements to find matches
-        items = []
+        items = MediaContainer[cls](self._server, data, initpath=initpath) if data.tag == 'MediaContainer' else []
```

Comment:
`AttributeError: 'list' object has no attribute 'tag'` when `data = []` from the line above.

Reply:
Maybe we can initialise an empty media container when the tag attribute does not exist? I'll create a pr soon if not already done.

Reply:
Initialize an `empty` element like what you did here:

https://github.com/pkkid/python-plexapi/blob/d6754c0cdeec67056df10318e708ca6ec4a61bb0/plexapi/base.py#L459

Reply:
should be resolved in #1396